packages/utils/src/date/index.ts

```diff
@@ -175,10 +179,45 @@ export function timeZoneExists(timeZone: string): boolean {
 	}
 }
 
+/**
+ * Date format options
+ * @enum string
+ */
+export const FORMAT = {
+	/** en: June 29, 2021 / fr: 29 juin 2020 / ja: 2020年6月29日 */
+	MDY_LONG: 'MDY_LONG',
+	/** en: June 2020 / fr: juin 2020 / ja: 2020年6月 */
+	MY_LONG: 'MY_LONG',
+	/** en: 06/29/2020 / fr: 29/06/2020 / ja: 2020/06/29 */
+	MDY: 'MDY',
+	/** en: 6/29/20, 10:00 PM / fr: 29/06/2020 22:00 / ja: 2020/06/29 22:00 */
+	MDYHM: 'MDYHM',
+};
+
+const options: Map = {
+	[FORMAT.MDY_LONG]: { year: 'numeric', month: 'long', day: 'numeric' },
+	[FORMAT.MY_LONG]: { year: 'numeric', month: 'long' },
+	[FORMAT.MDY]: { year: 'numeric', month: '2-digit', day: '2-digit' },
+	[FORMAT.MDYHM]: { dateStyle: 'short', timeStyle: 'short' },
```

Comment:
Seems like `timeStyle` in not part of the expected options object format https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_es5_d_.intl.datetimeformatoptions.html

Reply:
But hey are available in the [MDN doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat) 🤔